.gitignore

```diff
@@ -18,6 +19,8 @@
 .env.test.local
 .env.production.local
 .env
+package-lock.json
+yarn.lock
```

Comment:
Hello @ Jayakumar03, you miss understood, you don't have to put in the .gitignore file.

We recommend using `yarn` for development purposes so you should not be committing the `package-lock.json`.

Also, as you have not installed any new package so `yarn. lock` should not have any changes.

Please remove the `package-lock.json` and undo the changes to `yarn.lock` file.


```
 package-lock.json
yarn.lock
```